golden/golden_test.go

```diff
@@ -71,14 +80,21 @@ func TestGoldenAssert(t *testing.T) {
 	assert.True(t, success)
 }
 
-func setupGoldenFile(t *testing.T, content string) (string, func()) {
-	_ = os.Mkdir("testdata", 0755)
-	f, err := ioutil.TempFile("testdata", "")
+func setupGoldenFile(t *testing.T, relative bool, content string) (string, func()) {
```

Comment:
Maybe instead of the new boolean flag parameter the new test could just use `fs.NewFile()` ?

Reply:
Great idea!